package.json

```diff
@@ -56,8 +57,12 @@
     "@types/react": "^18.2.79",
     "@types/react-dom": "^18.2.25",
     "@vitejs/plugin-react": "^4.2.1",
+    "axe-playwright": "^2.0.1",
     "eslint": "^9.0.0",
-    "eslint-plugin-react": "^7.34.1",
+    "eslint-plugin-import": "^2.29.1",
```

Comment:
These eslint plugins are installed but they aren't imported anywhere, correct?

Reply:
there aren´t imported they were added to the package.json during the installation of the addon-a11y